tests/unit/gtReconf.cpp

```diff
@@ -68,7 +68,7 @@ TEST(LibopaecReconfCommonMOCKHW, gbs_reconf_01) {
 	fpga_token tok = &_tok;
 
 
-	EXPECT_EQ(FPGA_INVALID_PARAM, set_afu_userclock(h, usrlclock_high, usrlclock_low));
+	//EXPECT_EQ(FPGA_INVALID_PARAM, set_afu_userclock(h, usrlclock_high, usrlclock_low));

```

Comment:
❓

Reply:
This portion of the test case is invalid and was causing a segfault due to the uninitialized handle h.  Isaac hit this too, and filed an issue on it.